PRINCIPLES.md

```diff
@@ -46,8 +46,8 @@ for data in a different projection.
 
 - **Working code required.** Proposed changes should be accompanied by working code
 (ideally with a link to an online service running the code). A reference implementation should be available
-online to power the interactive documentation. Fully accepted specifications should have at least 3 implementations
```

Comment:
Not sure what "fully  accepted" means here, since 3 impls is what we call Candidate in the Maturity Classification model. Rewording to just *not* be inconsistent with the MC.

Reply:
Yeah, I think this predates extensions, and I think fully accepted was meant to mean like 1.0.0? Change looks good.